js/localization/messages/pt-BR.json

```diff
@@ -195,38 +195,38 @@
         "dxScheduler-confirmRecurrenceEditOccurrence": "Editar compromisso",
         "dxScheduler-confirmRecurrenceDeleteOccurrence": "Eliminar compromisso",
         "dxScheduler-noTimezoneTitle": "Sem fuso horário",
-        "dxScheduler-moreAppointments": "",
+        "dxScheduler-moreAppointments": "{0} +",
```

Comment:
Could we use "{0} mais" here?

Reply:
@BingoRUS Yes, that is another option for this translation. Let's use your suggestion, but both are valid. I will change it here.